runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -189,6 +189,11 @@ public void startUp()
 		chatCommandManager.registerCommandAsync(PB_COMMAND, this::personalBestLookup, this::personalBestSubmit);
 		chatCommandManager.registerCommandAsync(GC_COMMAND_STRING, this::gambleCountLookup, this::gambleCountSubmit);
 		chatCommandManager.registerCommandAsync(DUEL_ARENA_COMMAND, this::duelArenaLookup, this::duelArenaSubmit);
+
+		chatCommandManager.registerCommandAlias(TOTAL_LEVEL_COMMAND_STRING, "!overall");
+		chatCommandManager.registerCommandAlias(PRICE_COMMAND_STRING, "!pc");
+		chatCommandManager.registerCommandAlias(LEVEL_COMMAND_STRING, "!level");
+		chatCommandManager.registerCommandAlias(CMB_COMMAND_STRING, "!cb", "!cblvl");
```

Comment:
In my opinion, this would be better and simpler as either (a) invoking `registerCommand(Async)?` with the alternative command strings, or (b) changing the command registration API to accept some array or collection of strings.

Reply:
I agree with that, I thought about option b before but wasn't sure how to go about where the alias would be. Maybe changing the command string constant to an array. When I can, I'll make the changes and see how it goes

Reply:
I'd vote on just registering the command twice probably.